FirebaseSegmentation/Sources/SEGDatabaseManager.m

```diff
@@ -174,13 +174,13 @@ - (void)createOrOpenDatabaseWithCompletion:(SEGRequestCompletion)completionHandl
 }
 
 - (void)removeDatabase:(NSString *)path completion:(SEGRequestCompletion)completionHandler {
-  __weak SEGDatabaseManager *weakSelf = self;
   dispatch_async(_databaseOperationQueue, ^{
-    SEGDatabaseManager *strongSelf = weakSelf;
+    SEGDatabaseManager *strongSelf = self;
```

Comment:
You can remove this declaration (and the if statement below) and use `self` directly.